### PR TITLE
CHE-10544: prevent Dashboard to become broken after deleting a workspace

### LINKED ISSUE #10544: Deleting a not supported workspace breaks dashboard in OpenShift
### Description
If you create a workspace that is not supported in a particular spin of Che and then delete it, the workspaces tab fails to populate with error `Failed to retrieve workspace wksp-o078 data. Reason: Workspace with id 'workspacenn25oh2umzhkbgt9' doesn't exist` until the page is refreshed. On the dashboard tab the deleted workspace is still listed.

### Reproduction Steps
1. Deploy current snapshot of Che on minishift
2. Create Java-MySQL workspace, choose "Edit" option (or click top "Create" button)
3. See that this workspace stack is not supported, delete workspace
4. Che redirects to workspaces tab, which fails to load any workspaces. 
5. Dashboard tab still lists deleted workspace

**OS and version:**    
```
Che 6.9.0-SNAPSHOT (current master branch)
minishift v1.21.0+a8c8b37
oc v3.9.0+191fece
kubernetes v1.9.1+a0ce1bc657
features: Basic-Auth GSSAPI Kerberos SPNEGO

Server https://192.168.42.148:8443
openshift v3.9.0+46ff3a0-18
kubernetes v1.9.1+a0ce1bc657
```

### PR DESCRIPTION
**Pull request by akurinnoy**
### What does this PR do?
This PR fixes the bug when Dashboard becomes broken after deleting a workspace. It happens in case if a user creates a workspace and delete it immediately after redirecting to Workspace Details page.

### What issues does this PR fix or reference?
fix #10544

#### Release Notes

N/A - bugfix

#### Docs PR

N/A - bugfix

### CODE REVIEW

**Review by olexii4 — APPROVED** [review]

**Comment by sunix** [comment]
to me, header changes should be done in a separate PR

**Review by benoitf — APPROVED** [review]

**Comment by benoitf** [comment]
it's a separate commit so I'm fine as long as it remains as two commits

File: dashboard/Dockerfile [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
-# which accompanies this distribution, and is available at
-# http://www.eclipse.org/legal/epl-v10.html
+# Copyright (c) 2015-2018 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
```
Comment (benoitf) [inline dashboard/Dockerfile:8]: FYI I'm not sure we need the Contributors: section in EPL 2

Comment (akurinnoy) [reply dashboard/Dockerfile:8]: I've just updated these header as it has been done in `*.ts` files.
@skabashnyuk what do you think?

Comment (skabashnyuk) [reply dashboard/Dockerfile:8]: I'm not sure too

**Pull request merged**